packages/apps/kubernetes/templates/helmreleases/cilium.yaml

```
@@ -46,3 +50,5 @@ spec:
   - name: {{ .Release.Name }}
     namespace: {{ .Release.Namespace }}
   {{- end }}
+  - name: {{ .Release.Name }}-gateway-api
+    namespace: {{ .Release.Namespace }}
```

Let's make this configurable:
```suggestion
  {{- if $.Values.addons.gatewayAPI.enabled }}
  - name: {{ .Release.Name }}-gateway-api
    namespace: {{ .Release.Namespace }}
  {{- end }}
```